Contributing.md

```diff
@@ -2,24 +2,23 @@
-- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
-- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
-- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+* [Watch this video to get started, if you have no clue about open source](https://youtu.be/SL5KKdmvJ1U)
+* [Getting started with Git and GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+* [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+* [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+* [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
 
 ## 💥 Contribution
 
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
-[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=git&logoColor=white)](https://github.com/swapnilsparsh/Rescue/pulls)[![CodeFactor](https://www.codefactor.io/repository/github/vinitshahdeo/portscanner/badge)](https://www.codefactor.io/repository/github/swapnilsparsh/Rescue) [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/swapnilsparsh/Rescue)
 
-- Take a look at the Existing [Issues](https://github.com/swapnilsparsh/Rescue/issues) or create your own Issues!
-- Wait for the Issue to be assigned to you after which you can start working on it.
-- Fork the Repo and create a Branch for any Issue that you are working upon.
-- Read the [Code of Conduct](https://github.com/swapnilsparsh/Rescue/blob/master/Code_of_Conduct.md)
-- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
-- Add Screenshots to help us know what this Script is all about.
-- You can discuss with other people working on the same project about your idea on [discord.](https://discord.com/channels/731001496417730611/792257592322949121)
+* Take a look at the Existing [Issues](https://github.com/swapnilsparsh/Rescue/issues) or create your own Issues!
+* Wait for the Issue to be assigned to you after which you can start working on it.
+* Fork the Repo and create a Branch for any Issue that you are working upon.
+* Read the [Code of Conduct](https://github.com/swapnilsparsh/Rescue/blob/master/Code_of_Conduct.md)
+* Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+* Add Screenshots to help us know what this Script is all about.
+* You can discuss with other people working on the same project about your idea on [discord.](https://discord.com/channels/731001496417730611/792257592322949121)
 
```

Comment:
Please add these guidelines also under the guidelines section.

- Participants have to comment on issues they would like to work on, and mentors will assign you.
- Issues will be assigned on a first-come, first-serve basis.
- Participants can also open their issues, but it needs to be verified by a mentor.
- Each participant will be assigned 2 issues (max) at a time to work.
- Participants/contributors have max 4 days to complete issues. After that issues will be assigned to others.
- Participants are expected to follow project guidelines and coding style.
- It might take a maximum of 2 days to review your pull request. Please have patience.